packages/fhir-care-team/src/components/ListView/tests/index-rtl.test.tsx

```diff
@@ -162,18 +168,20 @@ describe('Care Teams list view', () => {
 
     const viewDetailsLink = getByText(/View Details/);
     expect(viewDetailsLink).toMatchInlineSnapshot(`
-      <a
-        href="/admin/CareTeams/308"
-      >
+      <span>
         View Details
-      </a>
+      </span>
     `);
     fireEvent.click(viewDetailsLink);
-    expect(history.location.pathname).toEqual('/admin/CareTeams/308');
```

Comment:
`    expect(history.location.pathname).toEqual('/admin/CareTeams/308');`
Why was this removed.

Reply:
Instead of the path, we're testing for the querystring that has been added to the path, hence replaced pathname with search.

Reply:
I get that, but is the previous assertion no longer necessary, why was it there initially

Reply:
Initially, view-details was just creating a path using the care team url plus the id of the care team hence getting the pathname  to match was the assertion.